e2e/handles/handles.test.ts

```diff
@@ -58,84 +55,95 @@ describe("🤝 Handles", () => {
             await ExtrinsicHelper.runToBlock(currentBlock + expirationOffset + 1); // Must be at least 1 > original expiration
 
             const retireHandle = ExtrinsicHelper.retireHandle(msaOwnerKeys);
-            const { target: event } = await retireHandle.fundAndSend(fundingSource);
+            const { target: event } = await retireHandle.signAndSend('current');
             assert.notEqual(event, undefined, "retireHandle should return an event");
             const handle = event!.data.handle.toString();
             assert.notEqual(handle, "", "retireHandle should return the correct handle");
         });
     });
 
-    describe("@Alt Path: Claim Handle with possible presumptive suffix/RPC test", () => {
```

Comment:
This was just moved into a separate describe block so that it doesn't interact with the ones above.